doc/api/console.md

```diff
@@ -418,7 +418,9 @@ added: v0.1.104
 Starts a timer that can be used to compute the duration of an operation. Timers
 are identified by a unique `label`. Use the same `label` when calling
 [`console.timeEnd()`][] to stop the timer and output the elapsed time in
-milliseconds to `stdout`. Timer durations are accurate to the sub-millisecond.
+suitable timeunits to `stdout`. For example, if the elapsed
+time is 3869ms, `console.timeEnd()` now displays "3.869s". Timer
+durations are accurate to the sub-millisecond.
```

Comment:
Small side note, is this supposed to read “to the millisecond”? It doesn’t really make sense to say that there might be more granularity than that if the timer doesn’t actually expose it, right?

Reply:
@yashLadha can you please also fix this?

Reply:
Done 👍  Removed the un-necessary granularity details.